wesnoth-thot.po

```diff
@@ -1,5 +1,5 @@
-# Vladimír Slávik <slavik.vladimir@seznam.cz>, 2007 - 2011.
-# Petr Šlejhar (Septim) <septim@centrum.cz>, 2008 - 2011.
+# Vladimír Slávik <slavik.vladimir@seznam.cz>, 2008 - 2011.
+# Petr Šlejhar (Septim) <septim@centrum.cz>, 2008 - 2011, 2020 - 2022.
```

Comment:
Septime, každopádně dík, že jste se pustil do přidání sebe sama do hlaviček, mě jste tam už dlouho chyběl a nedávno jsem si dal do plánu vás tam přidat sám.

Jen bych rád udržoval v hlavičkách informaci o tom, že daný autor v daném roce do daného úseku hry sáhl.

Tak dlouho si nepamatuji, ale mám git a v něm nemůžu najít žádný váš příspěvek do THoT za rok 2021.

Prosím, uveďte mne na pravou míru, pokud se mýlím. V opačném případě prosím strpte mé opravy.
```suggestion
# Petr Šlejhar (Septim) <septim@centrum.cz>, 2008 - 2011, 2020, 2022.
```

Reply:
To je možné, bral jsem toto své druhé období vcelku a je pravda, že jsem nekontroloval, jestli skutečně mám nějaký příspěvek v konkrétním roce. Stejně to předchozí období beru ze zachovalé mailové konverzace a nejsem si úplně jist, kolik řetězců jsem v který rok posílal (a jestli vůbec nějaké). Vím jen, že jsme se v tom období tou kampaní zabývali. Ale jestli Vláďa posílal něco ještě v roce 2011, tak to bylo určitě ode mě (kampaň jsem přeložil a pak překlad udržoval).

Reply:
V tomto sledu jsem posílal věci, u kterých jsem si byl jistější (a je vidět, že ne vždy to 100% platilo). U ostatních katalogů jsem toho taky dost dělal (nebylo výjimkou, že jsem posílal hromadu drobných úprav třeba do pěti katalogů), ale to už nedám dohromady. Ale bylo to někdy období 2008 až 2011. A samozřejmě v nějakém gitu nedohledatelné, neboť vše šlo přes mail a přidával Vláďa či Oto.

Reply:
Vím jen, že nebyla kampaň, do které bych tenkrát něco nedělal. U těch technických katalogů si nejsem jist, jen ty úplně technické („ajťácké“) jsem nedělal určitě.

Reply:
Dík za poodhalení roušky minulosti. Je mi jasné, že toho víte víc než upstream git, kam posílal správce, mimochodem [vaše jméno se objevilo o pár dní později](https://github.com/wesnoth/wesnoth/blob/6f73f976c7cbcf71b66d53989d2ff5940fac2ed8/po/wesnoth-thot/cs.po). A protože vím, jak pracujete, nepochybuji že i tenkrát jste byl oporou týmu.

Nicméně tu změnu Vládi z 2007 na 2008 zatím nevidím podloženou a za rok 2021 v THoT se také neperete. Můžeme se domluvit takto?

- nebudete měnit roky ostatním, pokud nebudete mít důkaz, že je tam chyba
- můžete si změnit staré roky (okolo 2010), jestliže víte, že jste byl někde opomenut
- vaše nové roky dám s vaším souhlasem dohromady já z našeho gitu, klidně v rámci stávajících PR

Reply:
OK, to zní rozumně.
Jen pro vysvětlení, já mám z konference uložené jen maily, které obsahovaly nějakou zajímavou informaci nebo tip navíc apod. Žádná ta klasická sdělení typu tady posílám překlady či odpovědi typu tvé překlady byly zkontrolovány a zařazeny. Proto mám tu starou konverzaci poměrně troskoidní. Vycházel jsem tedy často i ze svých vzpomínek, kde bylo např. uloženo, že jsem THoT překládal z nuly. Nejde mi vůbec o nějaké roky více nebo méně (to jako fakt ne), jen mi to přišlo, že je to tak správně.